packages/labs/cli/src/lib/init/element-starter/templates/gitignore.ts

```diff
@@ -0,0 +1,13 @@
+import {FileTree} from '@lit-labs/gen-utils/lib/file-utils.js';
+import {Language} from '../../../commands/init.js';
+
+export const generateGitignore = (lang: Language): FileTree => {
+  return {
+    '.gitignore': `node_modules${
```

Comment:
Is a use case running this in an existing repo? If so we probably shouldn't overwrite `.gitignore`. If not, we should maybe start with only generating a new folder so we know it's clean.

Reply:
It sure seems like that's a good use case, but maybe that should be a separate command?